src/components/FilterCollapse.js

```diff
@@ -338,6 +334,7 @@ class FilterCollapse extends Component {
     if (!valid) return;
 
     if (tokens.length > 0) {
+      Query.set('f', this.state.filterInput);
```

Comment:
Why this change?

Reply:
Before we were updating the query as the user typed. I changed it so the query string is only ever what the user should be seeing at that moment.